src/test/e2e/api/admin/api-token.auth.e2e.test.ts

```diff
@@ -207,28 +207,32 @@ test('Token-admin should be allowed to create token', async () => {
 test('An admin token should be allowed to create a token', async () => {
     expect.assertions(2);
 
-    const adminToken = await db.stores.apiTokenStore.insert({
-        type: ApiTokenType.ADMIN,
-        secret: '12345',
-        environment: '',
-        projects: [],
-        tokenName: 'default-admin',
-    });
+    const { request, destroy, services } = await setupAppWithAuth(stores);
 
-    const { request, destroy } = await setupAppWithAuth(stores);
+    const { secret } =
+        await services.apiTokenService.createApiTokenWithProjects(
+            {
+                tokenName: 'default-admin',
+                type: ApiTokenType.ADMIN,
+                projects: ['*'],
+                environment: '*',
+            },
+            'test',
+            1,
+        );
```

Comment:
This uses the service logic that also pushes the newly created token in the memory cache